tests/python/pants_test/util/test_meta.py

```diff
@@ -4,9 +4,18 @@
 import unittest
 from abc import ABC, abstractmethod
 from dataclasses import FrozenInstanceError, dataclass
+from enum import Enum
```

Comment:
What are these imports for?

Reply:
The pre-commit hook doesn't check for all changes since master, just staged changes!